datatypes/src/commonMain/kotlin/at/asitplus/crypto/datatypes/CryptoSignature.kt

```diff
@@ -88,6 +88,10 @@ sealed interface CryptoSignature : Asn1Encodable<Asn1Element> {
     ) : CryptoSignature {
 
         override val signature: Asn1Element = Asn1.Sequence { +r.encodeToTlv(); +s.encodeToTlv() }
+        init {
+            require(r.isPositive) { "r must be positive" }
+            require(s.isPositive) { "s must be positive" }
+        }
```

Comment:
simple sanity check, since much of our encoding logic depends on it

Reply:
GitHub doesn't show it, but be sure to add an `@Throws` annotation to the constructor